webpack.config.js

```diff
@@ -22,29 +22,17 @@ const sharedConfig = {
 		filename: '[name].js',
 		chunkFilename: '[name].js',
 	},
-	module: {
-		...defaultConfig.module,
-		rules: [
-			...defaultConfig.module.rules.map(
-				( rule ) => {
-					// @todo Can remove once the default config no longer excludes excludes CSS related to Gutenberg components.
-					if ( rule.test.source === '\\.css$' ) {
-						rule.exclude = /node_modules\/(?!@wordpress)/;
+	plugins: [
+		...defaultConfig.plugins
+			.map(
+				( plugin ) => {
+					if ( plugin.constructor.name === 'MiniCssExtractPlugin' ) {
+						plugin.options.filename = '../css/[name].css';
 					}
-					return rule;
+					return plugin;
 				},
-			),
-		],
-	},
-	plugins: [
-		...defaultConfig.plugins.map(
-			( plugin ) => {
-				if ( plugin.constructor.name === 'MiniCssExtractPlugin' ) {
-					plugin.options.filename = '../css/[name].css';
-				}
-				return plugin;
-			},
-		),
+			)
+			.filter( ( plugin ) => plugin.constructor.name !== 'CleanWebpackPlugin' ),
```

Comment:
The `CleanWebpackPlugin` was [added](https://github.com/WordPress/gutenberg/pull/23135/files) recently to the default Webpack config we import.

I've decided to remove the plugin from the shared config as it was causing some build failures. In fact, it currently doesn't fit our current setup, as having a shared `output.path ` for the multiple Webpack configurations we have leads to the build folder being cleared multiple times during compilation (which means some assets are deleted as soon as they are compiled). There is a potential [solution](https://github.com/johnagan/clean-webpack-plugin/issues/157#issuecomment-518347683) for this, but I don't think it's worth the hassle.

Reply:
I defer to @johnwatkins0

Reply:
Sounds good to me.